framework/src/postprocessors/GreaterThanLessThanPostprocessor.C

```diff
@@ -86,13 +86,13 @@ GreaterThanLessThanPostprocessor::execute()
 
   if (_comparator == "greater")
   {
-    for (auto dof : aldit._all_dof_indices)
+    for (auto dof : aldit.getDofIndices())
       if (solution(dof) > _value)
         ++_count;
   }
   else if (_comparator == "less")
   {
-    for (auto dof : aldit._all_dof_indices)
+    for (auto dof : aldit.getDofIndices())
```

Comment:
If `aldit.getDofIndices` is the whole world, why we did not see seg-fault here.

`solution` is serialized?

Reply:
It's not the whole world. It's only local dofs. We did not call `dofIndicesSetUnion`